crates/config/maximal-config-example.toml

```diff
@@ -124,6 +124,11 @@ max_receipt_value_grt = "0.001" # 0.001 GRT. We use strings to prevent rounding
 # max_amount_willing_to_lose_grt = "0.1"
 max_amount_willing_to_lose_grt = 20
 
+# List of Gateways that are allowed to spend up to `max_amount_willing_to_lose_grt`
+# over the escrow balance
+trusted_gateways = ["0xdeadbeefcafebabedeadbeefcafebabedeadbeef"]
```

Comment:
Can we use the term "sender", rather than introducing the term "gateway" unnecessarily?

Reply:
good catch!